docs/apis.md

```diff
@@ -90,7 +90,7 @@ The [Check API](./apis/subsystems/check/index.md) allows you to add security, pe
 
 ### Comment API (comment)
 
-The [Comment API](https://docs.moodle.org/dev/Comment_API) allows you to save and retrieve user comments, so that you can easily add commenting to any of your code.
+The [Comment API](./api/core/comment) allows you to save and retrieve user comments, so that you can easily add commenting to any of your code.
```

Comment:
Relative links should be to a file, and this should link to `apis` not `api`.
```suggestion
The [Comment API](./apis/core/comment/index.md) allows you to save and retrieve user comments, so that you can easily add commenting to any of your code.
```